File: SWexpert/D1/2068/2068.java
import java.util.Scanner;

public class Solution{
    public static void main(String[] args){
        int t, m;
        Scanner s = new Scanner(System.in);
    	t = s.nextInt();
    
    	for(int j = 0; j < t; j++){
        	m = 0;
	    	for(int i = 0; i < 10; i++){
	            int temp;
	            temp = s.nextInt();
                if(m < temp){
                    m = temp;
                }
	    	}
	        System.out.printf("#%d %d\n", j+1, m);
	    }
    }
}

File: SWexpert/D1/2068/2068.cpp
#include <iostream>
using namespace std;

int main(void){
    int t, m;
    cin >> t;
    
    for(int j = 0; j < t; j++){
        m = 0;
	    for(int i = 0; i < 10; i++){
            int temp;
            cin >> temp;
            ( m > temp ) ? m : ( m = temp );
    	}
        cout << "#" << j + 1 << " " << m << endl;
    }
    return 0;
}
